release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2616,6 +2616,10 @@ This issue affects the CPU load balancing features because these features depend
 
 * In hosted control planes for {product-title}, using the {secrets-store-operator} with AWS Secrets Manager or AWS Systems Manager Parameter Store is not supported. (link:https://issues.redhat.com/browse/OCPBUGS-18711[*OCPBUGS-18711*])
 
+* In hosted control planes for {product-title}, the `default`, `kube-system`, and `kube-public` namespaces are not properly excluded from pod security admission. (link:https://issues.redhat.com/browse/OCPBUGS-21776[*OCPBUGS-21776*])
+
+* In hosted control planes for {product-title}, created namespaces are incorrectly enforcing the `restricted` pod security admission profile instead of the `privileged` profile. (link:https://issues.redhat.com/browse/OCPBUGS-20526[*OCPBUGS-20526*])
```

Comment:
`OCPBUGS-20526` has just submitted PR targeting 4.14 and becomes ON_QA now. Need wait if QE verification can pass.

Reply:
@xingxingxia Are you saying that if https://issues.redhat.com/browse/OCPBUGS-20526 passes verification in the next day or so, then we can just remove this from the known issues list completely?

Reply:
And just in case it doesn't pass in time, let me know if the wordings in this PR look okay as is or not, thanks!

Reply:
@bergerhoffer
https://issues.redhat.com/browse/OCPBUGS-20526 is fixed, I think we can remove this.

And the (link:https://issues.redhat.com/browse/OCPBUGS-21776[*OCPBUGS-21776*]) is targeted to OCP 4.15. This can be replaced with https://issues.redhat.com/browse/OCPBUGS-22379,  the PR is still open. 
(The fix for OCP 4.15 is mergerd, will be verified by tomorrow)

Reply:
Thanks @gkarager!

I've removed the entry for OCPBUGS-20526.

I've also replaced the link to OCPBUGS-21776 with OCPBUGS-22379. Let me know whether it gets verified in time for GA (and we can just close this PR) or if it doesn't and we need to merge this PR in. Thanks!